### PR TITLE
java-openliberty: add stack image label

### PR DESCRIPTION
**Pull request by awisniew90**
### Checklist:

- [ x] Read the [Code of Conduct](https://github.com/appsody/website/blob/master/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md).

- [ x] Followed the [commit message guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md#commit-message-guidelines).

- [ x] Stack adheres to [Appsody stack structure](https://github.com/appsody/website/blob/master/content/docs/stacks/stacks-overview.md#stack-structure).

### Modifying an existing stack:

- [x ] Updated the stack version in `stack.yaml`

Add image labels to Dockerfile-stack image

### CODE REVIEW

File: incubator/java-openliberty/image/Dockerfile-stack [hunk 1]
```diff
@@ -1,4 +1,11 @@
 FROM kabanero/ubi8-maven:0.3.1
+
+LABEL vendor="Open Liberty" \
+    name="appsody/java-openliberty" \
```
Comment (neeraj-laad) [inline incubator/java-openliberty/image/Dockerfile-stack:4]: We could use {{.stack.id}} for this?

File: incubator/java-openliberty/image/Dockerfile-stack [hunk 2]
```diff
@@ -1,4 +1,11 @@
...
+    version="{{.stack.version}}" \
+    summary="Image for java-openliberty development" \
```
Comment (neeraj-laad) [inline incubator/java-openliberty/image/Dockerfile-stack:6]: {{.stack.name}} ?

File: incubator/java-openliberty/image/Dockerfile-stack [hunk 3]
```diff
@@ -1,4 +1,11 @@
...
+    description="This image is the development container for the java-openliberty stack"
```
Comment (neeraj-laad) [inline incubator/java-openliberty/image/Dockerfile-stack:7]: {{.stack.description}} ?

**Comment by awisniew90** [comment]
@neeraj-laad  - updated the labels with stack variables like you suggested. Keeps things cleaner.

**Pull request merged**